### PR TITLE
Journal

### PR DESCRIPTION
**Pull request by OkumagbeHenoch**

### CODE REVIEW

File: components/Dialog.tsx [hunk 1]
```diff
@@ -17,78 +18,21 @@ export const showDialog = (
...
       {dialog}
-    </Dialog>,
+    </Dialog>
   );
+  
 
   document.body.append(container);
 };
 
-export const showAlertDialog = ({
```
Comment (epikoder) [inline components/Dialog.tsx:32]: Why remove a component that's in use?

**Comment by OkumagbeHenoch** [comment]
Ok nice

File: components/Dialog.tsx (hunk 1 above)
Comment (OkumagbeHenoch) [reply components/Dialog.tsx:32]: My bad.

Comment (OkumagbeHenoch) [reply components/Dialog.tsx:32]: > Why remove a component that's in use?

What do I have to install to run the frontend + backend on my system
not tampering with the code Ser ?

**Pull request merged**